pyanaconda/screen_access.py

```diff
@@ -93,12 +93,15 @@ def write_out_config_file(self, config_path=None):
 
         with self._lock:
             new_config_file = not os.path.exists(config_path)
-            with open(config_path, "wt") as f:
-                if new_config_file:
-                    # we are creating a new file, so add a header that it was created by Anaconda,
-                    # including its version number
-                    f.write(self._get_new_config_header())
-                self._config.write(f)
+            try:
+                with open(config_path, "wt") as f:
+                    if new_config_file:
+                        # we are creating a new file, so add a header that it was created by Anaconda,
+                        # including its version number
+                        f.write(self._get_new_config_header())
+                    self._config.write(f)
+            except OSError:
+                log.exception("Can't write user interaction config file.")
```

Comment:
The open() can generate quite a lot of various exceptions - so the general idea is to for now just to prevent the crash & log the exception. Later on (and possibly once we have more data) we can narrow down the scope of the except clause & log  specific warnings/errors.